### PR TITLE
fix: fix sso-tab

### PR DESCRIPTION
**Pull request by Yimin-Jin**
BUG: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/29256520

The reason I added the ajv dependency is that without it, I encounter the following error during startup:
微信截图_20240829151329

### CODE REVIEW

File: templates/ts/sso-tab-with-obo-flow/package.json.tpl [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "@microsoft/teams-js": "^2.22.0",
     "@microsoft/teamsfx": "^2.2.0",
     "@microsoft/teamsfx-react": "^3.0.0",
+    "ajv": "^8.17.1",
```
Comment (huimiu) [inline templates/ts/sso-tab-with-obo-flow/package.json.tpl:13]: Why we need those two dependencies?

Comment (Yimin-Jin) [reply templates/ts/sso-tab-with-obo-flow/package.json.tpl:13]: During my local debug, not including this dependency results errors. The detailed reason is writed in the description.

Comment (Yimin-Jin) [reply templates/ts/sso-tab-with-obo-flow/package.json.tpl:13]: What puzzles me is that no one else has reported this error. Is it possible that it only occurs on my local debug?

Comment (hund030) [reply templates/ts/sso-tab-with-obo-flow/package.json.tpl:13]: I think it's a bad practice to add any dependency that is not required in our source code.

Comment (Yimin-Jin) [reply templates/ts/sso-tab-with-obo-flow/package.json.tpl:13]: Okay, I'll remove these dependency.

**Review by hund030 — APPROVED** [review]

**Pull request merged**